spec/fixtures/users.yml

```diff
@@ -31,13 +37,23 @@ affiliate_manager:
   contact_name: Affiliate Manager
   welcome_email_sent: true
 
+affiliate_manager_with_pending_email_verification_status:
```

Comment:
Tried to rename but started to become a yak shaving processes so leaving it here for now.